### PR TITLE
fixes #90 - change to Apache 2.0 license

### LINKED ISSUE #90: relicense under Apache 2.0
just like e.g. https://github.com/scala/scala-xml/pull/242 and https://github.com/scala/scala-parser-combinators/pull/184

### PR DESCRIPTION
**Pull request by Sciss**
- changes source file headers
- changed LICENSE.md -> LICENSE with new text
- added NOTICE and docs/licenses/
- moved some images from examples from source directory to resources
directory

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+scala-swing
+
+Copyright (c) 2002-2019 EPFL
+Copyright (c) 2011-2019 Lightbend, Inc.
+Copyright (c) 2019 contributors
```
Comment (SethTisue) [inline NOTICE]: not sure about this line. are there legal reasons to add it, or not to add it? or is it more of a credit-where-credit-is-due kind of thing? if you want to include it, I probably need to check first with the Scala Center and Lightbend to see whether it's okay. I have no objection in principle, I think we just need to know what the implications are, if any.

Comment (SethTisue) [reply NOTICE]: especially in light of https://github.com/scala/scala-swing/issues/68

Comment (Sciss) [reply NOTICE]: Technically everyone that commits code creates a copyrighted work (cf https://en.wikipedia.org/wiki/Berne_Convention), or in the case of a collaborative work, contributes to its copyright, in my understanding. And yes, it is also crediting the work.

Comment (Sciss) [reply NOTICE]: Re CLA: "The purpose of a CLA is to ensure that the guardian of a project's outputs has the necessary ownership or grants of rights over all contributions to allow them to distribute under the chosen license"; so this is precisely to guarantee that code owned by the person that contributes it, can be unrestrictedly used under the project's license.

Comment (Sciss) [reply NOTICE]: E.g. https://euipo.europa.eu/ohimportal/en/web/observatory/faqs-on-copyright-de#2

Comment (SethTisue) [reply NOTICE]: okay. I guess it's not a problem considering the "Grant of Copyright License" clause of the Apache 2 license

**Comment by SethTisue** [comment]
fyi @adriaanm

**Comment by SethTisue** [comment]
thank you, @Sciss, for taking this on.

File: NOTICE (hunk 1 above)
Comment (SethTisue) [reply NOTICE]: discussion on this at https://github.com/typelevel/cats-effect/issues/521

for the record, I'm not too concerned about it in the case of scala-swing, but we might hesitate before following suit in any other repos

**Pull request merged**